### PR TITLE
Rahulbcn27 GitHub actions 1

### PR DESCRIPTION
**Pull request by rahulbcn27**
Migration from Travis CI to Github Actions CI.

### CODE REVIEW

**Comment by rahulbcn27** [comment]
@rohityadavcloud Please review.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Build
```
Comment (rohityadavcloud) [inline .github/workflows/build.yml:1]: @rahulbcn27 missing license?

**Review by yadvr — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (rahulbcn27) [reply .github/workflows/build.yml:18]: Fixed.

**Review by yadvr — APPROVED** [review]
LGTM - only way to test is to merge this. Merging this.

**Review by yadvr — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
Feedback @rahulbcn27 write better summary in your commit (i.e. commit message) and PR title/description.

**Pull request merged**